src/Console/ConfigurationResolver.php

```diff
@@ -784,4 +784,36 @@ private function setOption($name, $value)
 
         $this->options[$name] = $value;
     }
+
+    /**
+     * @param string $optionName
+     *
+     * @return bool
+     */
+    private function resolveOptionBooleanValue($optionName)
+    {
+        $value = $this->options[$optionName];
+        if (is_bool($value)) {
+            return $value;
```

Comment:
is it ever happening, as we pass value of InputOption which is `yes`/`no` string ?